src/core/pagerankGraph.test.js

```diff
@@ -165,6 +166,131 @@ describe("core/pagerankGraph", () => {
     });
   });
 
+  describe("edge filtering", () => {
+    const src1 = NodeAddress.fromParts(["src", "1"]);
+    const src2 = NodeAddress.fromParts(["src", "2"]);
+    const dst1 = NodeAddress.fromParts(["dst", "1"]);
+    const dst2 = NodeAddress.fromParts(["dst", "2"]);
+    const e11 = {
+      src: src1,
+      dst: dst1,
+      address: EdgeAddress.fromParts(["e", "1", "1"]),
+    };
+    const e12 = {
+      src: src1,
+      dst: dst2,
+      address: EdgeAddress.fromParts(["e", "1", "2"]),
+    };
+    const e21 = {
+      src: src2,
+      dst: dst1,
+      address: EdgeAddress.fromParts(["e", "2", "1"]),
+    };
+    const e22 = {
+      src: src2,
+      dst: dst2,
+      address: EdgeAddress.fromParts(["e", "2", "2"]),
+    };
+    const graph = () =>
+      [e11, e12, e21, e22].reduce(
+        (g, e) => g.addEdge(e),
+        [src1, src2, dst1, dst2].reduce((g, n) => g.addNode(n), new Graph())
+      );
+
+    const pagerankGraph = () => new PagerankGraph(graph(), defaultEvaluator);
+
+    function expectPagerankGraphToEqualGraph(options: EdgesOptions | void) {
+      const pagerankGraphEdges = Array.from(
+        pagerankGraph().edges(options)
+      ).sort();
+      const graphEdges = Array.from(graph().edges(options)).sort();
+
+      const edgesAreEqual = (e1, e2) =>
+        e1.edge.address === e2.address &&
+        e1.edge.src === e2.src &&
+        e1.edge.dst === e2.dst;
+
+      pagerankGraphEdges.forEach((pgEdge, i) =>
+        expect(edgesAreEqual(pgEdge, graphEdges[i])).toBe(true)
+      );
+    }
+
+    describe("edge filter matches graph edge filter", () => {
+      it("finds all edges when no options are specified", () => {
+        expectPagerankGraphToEqualGraph(undefined);
+      });
+      it("finds all edges when universal filters are specified", () => {
```

Comment:
this isn't a universal filter, it's a filter that gets every edge in the graph.
a universal filter would be:
```
{
addressPrefix: EdgeAddress.empty,
srcPrefix: NodeAddress.emtpy,
dstPrefix: NodeAddress.empty,
}
```

Reply:
The thesaurus turned up `all-inclusive`.. that feels like a precise description.